app/views/organisations/edit.html.erb

```diff
@@ -0,0 +1,40 @@
+<% content_for :title, "Change #{@organisation.name}'s details" %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+    text: 'Back',
+    href: :back,
+  ) %>
+<% end %>
+
+<%= form_for(@organisation, as: :organisation, html: { method: :patch }) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">
+        <%= content_for(:title) %>
+      </h1>
+
+      <%= f.govuk_text_field :name,
+        autocomplete: "name"
+      %>
+
+      <%= f.govuk_text_field :address_line1,
+        autocomplete: "address-line1"
+      %>
+
+      <%= f.govuk_text_field :address_line2,
+        autocomplete: "address-line2"
```

Comment:
Again, not tested this, but I think this should work:

```suggestion
        label: { text: "Building and street line 2 of 2", hidden: true },
        autocomplete: "address-line2"
```

Output should be a visually hidden label that says ‘Building and street line 2 of 2’

Reply:
Do you not want a visible label here? It also sort of includes city at the moment?

Reply:
Do the addresses that we have, have a standard format. Probably not.

Perhaps the more fail safe version, as much as I hate it, is just to have:

```
Address line 1
[                            ]

Address line 2
[                            ]
```

Reply:
Not currently. I'm not sure if the old CORE system had any rules around how org addresses had to be saved. We'd need to see some data to check that I suppose so for now here it's just Line 1 and Line 2.